monailabel/utils/others/generic.py

```diff
@@ -223,3 +226,82 @@ def is_openslide_supported(name):
     if ext and ext in supported_ext:
         return True
     return False
+
+
+def get_bundle_models(app_dir, conf, conf_key="models"):
```

Comment:
Please use the MONAI bundle APIs instead of `model_info.json`:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/scripts.py#L362
@yiheng-wang-nv can help share more details if you want.
I think I put some similar comments in other PRs before.

Thanks.

Reply:
Hi @Nic-Ma , I saw the comment, thanks so much. But the functionalities are bits of different here. We can use the mentioned MONAI bundle API to get lists, but this get_bundle_models function is more adapt to MONAI Label, using some monailabel command configs, such as "models" "bundle", it also include some filtering logics as not all bundle should be returned to as available.

Thanks

Reply:
Hi @tangy5 ,

I don't mean to remove all the function logic here, I mean the `model_info.json` is private file in the model-zoo, not for public access.
Please use the APIs to get model information and do the model-label specific logic.

Thanks.

Reply:
I see, got the point, thanks, will use the API to replace the JSON file.

Reply:
Hi @tangy5 , feel free to ask if you meet any issue in using the API Nic suggests. BTW, the arg `auth_token` is important if frequently call the API since github sets different rate limit for authorized and unauthorized users.

Reply:
Done, thanks.